src/lib.rs

```diff
@@ -1,3 +1,22 @@
+#![warn(missing_docs)]
+//! # Motivation and Goals
+//!
+//! 1. Ergonomics is first priority. Being able to prototype quickly and code
```

Comment:
I think not having any is somewhat better than duplication, so it's fine to remove that.